submissions/krailoartur/HTML_CSS_practice/style.css

```diff
@@ -0,0 +1,186 @@
+*:focus {
+    outline-offset: 3px;
+    outline: 1px solid #74b9ff;
+}
+
+body {
+    color: #636e72;
+    background-color: #fff;
+    overflow-x: hidden;
+}
+
+a {
+    text-decoration: none;
+    color: inherit;
+}
+
+nav ul {
+    height: 70px;
+    margin: 0 0 0 0;
```

Comment:
In case like this use shorthand style: `margin: 0`, no need to write all these repeated numbers.